amy/workshops/forms.py

```diff
@@ -580,7 +580,8 @@ def clean_curricula(self):
                 missing_tags.add(tag)
         if missing_tags:
             raise forms.ValidationError(
-                f"You must add tags corresponding to these curricula. Missing tags: {', '.join(missing_tags)}"
+                f"""You must add tags corresponding to these curricula. """
+                f"""Missing tags: {', '.join(missing_tags)}"""
```

Comment:
In lines 583 and 584 you don't need to use `"""`. You can use `"` just fine.

Additionally, in L583 you defined an f-string, which is not used and will highlight in some linters. Drop `f` and you should be fine.